controller/lib/debug/trace.cpp

```diff
@@ -32,27 +32,26 @@ limitations under the License.
 #include <stdint.h>
 
 CircBuff<uint32_t, 0x4000> traceBuffer;
```

Comment:
May as well rename trace_buffer for consistency.  Same with traceVar array

Reply:
Done